source/dotnet/Library/AdaptiveCards/AdaptiveFallbackConverter.cs

```diff
@@ -2,10 +2,11 @@
 using Newtonsoft.Json.Linq;
 using System;
 using System.Collections.Generic;
+using System.Runtime.CompilerServices;
```

Comment:
is this needed?

Reply:
thanks

---
In reply to: [340735289](https://github.com/microsoft/AdaptiveCards/pull/3548#discussion_r340735289) [](ancestors = 340735289)